x/curating/types/params.go

```diff
@@ -73,7 +73,7 @@ func NewParams(
 
 // String implements the stringer interface for Params
 func (p Params) String() string {
-	out, _ := yaml.Marshal(p)
+	out, _ := yaml.Marshal(p) // nolint
```

Comment:
This instance was annoying. It was complaining about not checking the error, but it doesn't return an error type. I suppose we could modify the function to optionally return an error though...